_templates/page.html.erb

```diff
@@ -1,97 +1,24 @@
-  <script src="<%= File.join(javascripts_path, "bootstrap-offcanvas.js") %>" type="text/javascript"></script>
-  <%= render("_templates/_analytics.html.erb", :distro_key => distro_key) %>
-</head>
-<body>
-  <%= render("_templates/_topnav.html.erb", :distro_key => distro_key) %>
-  <div class="container">
-    <p class="toggle-nav visible-xs pull-left">
-      <button class="btn btn-default btn-sm" type="button" data-toggle="offcanvas">Toggle nav</button>
-    </p>
-    <ol class="breadcrumb">
-      <li class="sitename">
-        <a href="<%= site_home_path %>"><%= site_name %></a>
-      </li>
-      <li class="hidden-xs">
-        <%= breadcrumb_root %>
-      </li>
-      <li class="hidden-xs">
-        <%= breadcrumb_group %>
-      </li>
-      <%= breadcrumb_subgroup_block %>
-      <li class="hidden-xs active">
-        <%= breadcrumb_topic %>
-      </li>
-    </ol>
-    <div class="row row-offcanvas row-offcanvas-left">
-      <div class="col-xs-8 col-sm-3 col-md-3 sidebar sidebar-offcanvas">
-        <div class="row-fluid">
-          <%= render("_templates/_search.html.erb", :distro_key => distro_key) %>
-        </div>
-        <%= render("_templates/_nav.html.erb", :navigation => navigation, :group_id => group_id, :topic_id => topic_id, :subgroup_id => subgroup_id, :subtopic_shim => subtopic_shim) %>
-      </div>
-      <div class="col-xs-12 col-sm-9 col-md-9 main">
-        <div class="page-header">
-          <h1><%= article_title %></h1>
-        </div>
-        <%= content %>
-      </div>
-    </div>
-    <div class="visible-xs-block row row-offcanvas row-offcanvas-left">
-      <%= render("_templates/_footer.html.erb", :distro_key => distro_key, :images_path => images_path) %>
-    </div>
-  </div>
-  <script type="text/javascript">
-    /*<![CDATA[*/
-    $(document).ready(function() {
-      $("[id^='topicGroup']").on('show.bs.collapse', function (event) {
-        if (!($(event.target).attr('id').match(/^topicSubGroup/))) {
-          $(this).parent().find("[id^='tgSpan']").toggleClass("fa-angle-right fa-angle-down");
-        }
-      });
-      $("[id^='topicGroup']").on('hide.bs.collapse', function (event) {
-        if (!($(event.target).attr('id').match(/^topicSubGroup/))) {
-          $(this).parent().find("[id^='tgSpan']").toggleClass("fa-angle-right fa-angle-down");
-        }
-      });
-      $("[id^='topicSubGroup']").on('show.bs.collapse', function () {
-        $(this).parent().find("[id^='sgSpan']").toggleClass("fa-caret-right fa-caret-down");
-      });
-      $("[id^='topicSubGroup']").on('hide.bs.collapse', function () {
-        $(this).parent().find("[id^='sgSpan']").toggleClass("fa-caret-right fa-caret-down");
-      });
-    });
-    /*]]>*/
-  </script>
-  <%= render("_templates/_footer.html.erb", :distro_key => distro_key, :images_path => images_path) %>
-</body>
-</html>
+<%= render((distro_key['atomic'] ? "_templates/_page_atomic.html.erb" : "_templates/_page_atomic.html.erb"),
```

Comment:
> I think the second one here should be _page_openshift.html.erb

@Fryguy good catch. This is a workaround that I need help with.

Rendering the openshift page returns this error:

```
Error: Could not generate docs: undefined local variable or method `images_path' for #<AsciiBinder::TemplateRenderer:0x007f957b398e98>.
```
Full output: https://paste.fedoraproject.org/348201/14595147/ It may be a simple ruby issue I'm not aware of. Any ideas?